hello/lib/hello_web.ex

```diff
@@ -35,7 +35,8 @@ defmodule HelloWeb do
 
       # Import convenience functions from controllers
       import Phoenix.Controller,
-        only: [get_flash: 1, get_flash: 2, view_module: 1, view_template: 1]
+        only: [get_flash: 1, get_flash: 2, view_module: 1,
```

Comment:
@Vretamoza what is the benefit of using `only` when importing from a module?

Reply:
The benefit is that we get just the functionalities that we need instead of bringing all of them.